From e8571affa831677edde7503ec3178a22fc992de8 Mon Sep 17 00:00:00 2001
From: Kevin Chen <kevin_chen@aspeedtech.com>
Date: Wed, 21 May 2025 19:09:21 +0800
Subject: [PATCH] board: ast2600: Remove eSPI PC and VW initialization code

Leave OOB_FREE only. Move peripheral channel and virtual wire channel
into linux eSPI driver.

Change-Id: Ia57ba60e0b80859d4fdc934b104102aa58d0ce03
Signed-off-by: Kevin Chen <kevin_chen@aspeedtech.com>
(cherry picked from commit f04aa4b2e80bffeee76b8007cc4c18c7aad0836a)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 board/aspeed/evb_ast2600/evb_ast2600.c | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/board/aspeed/evb_ast2600/evb_ast2600.c b/board/aspeed/evb_ast2600/evb_ast2600.c
index 72ecb18c15..5a4ed7a33e 100644
--- a/board/aspeed/evb_ast2600/evb_ast2600.c
+++ b/board/aspeed/evb_ast2600/evb_ast2600.c
@@ -98,25 +98,6 @@ static void __maybe_unused espi_init(void)
 	 *
 	 * If you are not sure what you are doing, DO NOT use it.
 	 */
-	reg = readl(ESPI_BASE + 0x000);
-	reg |= 0xef;
-	writel(reg, ESPI_BASE + 0x000);
-
-	writel(0x0, ESPI_BASE + 0x110);
-	writel(0x0, ESPI_BASE + 0x114);
-
-	reg = readl(ESPI_BASE + 0x00c);
-	reg |= 0x80000000;
-	writel(reg, ESPI_BASE + 0x00c);
-
-	writel(0xffffffff, ESPI_BASE + 0x094);
-	writel(0x1, ESPI_BASE + 0x100);
-	writel(0x1, ESPI_BASE + 0x120);
-
-	reg = readl(ESPI_BASE + 0x080);
-	reg |= 0x50;
-	writel(reg, ESPI_BASE + 0x080);
-
 	reg = readl(ESPI_BASE + 0x000);
 	reg |= 0x10;
 	writel(reg, ESPI_BASE + 0x000);
-- 
2.34.1

